public/app/core/utils/UserProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { PureComponent } from 'react';
+import { getBackendSrv } from '@grafana/runtime';
+
+export interface Props {
```

Comment:
You probably need an `userId?: string` here at some point, as this `UserProvider` should also enable user retrieval.

Reply:
Yea, not there yet though. This is still just change password for the current user.